build-gluster-org/jobs/debian-ubuntu-package.yml

```diff
@@ -0,0 +1,57 @@
+- job:
+    name: debian-package-builder
+    node: master
```

Comment:
Use the label `debian10` for this job. We have a machine existing on jenkins https://build.gluster.org/computer/builder-deb10-1.int.rht.gluster.org/

Reply:
@deepshikhaaa done!

Reply:
@deepshikhaaa please validate the liburing-devel support as well.